.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test dev container
+
+on:
+  schedule:
+    - cron: "56 5 * * *"
+  workflow_dispatch:
+
+jobs:
+  dev-container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Test dev container
+        uses: devcontainers/ci@a56d055efecd725e8cfe370543b6071b79989cc8 # v0.3.1900000349
+        with:
+          runCmd: opensafely run run_all
+
+      # - name: Notify Slack on failure
+      #   if: failure() && github.event_name == 'schedule'
+      #   uses: zuplo/github-action-slack-notify-build@cf8e7e66a21d76a8125ea9648979c30920195552 # v2
+      #   env:
+      #     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+      #   with:
+      #     channel_id: <TODO: add a channel id>
+      #     status: "Scheduled dev container test run failure"
+      #     color: danger
```

Comment:
thought: A Slack notification is probably best for this kind of failure. This needs the channel choosing, and this section then being uncommented.